static/panes/output.ts

```diff
@@ -275,20 +275,20 @@ export class Output extends Pane<OutputState> {
     add(msg: string, lineNum?: number, column?: number, filename?: string) {
         const elem = $('<div/>').appendTo(this.contentRoot);
         if (lineNum) {
-            elem.html(
-                $('<span class="linked-compiler-output-line"></span>')
-                    .html(msg)
-                    .on('click', e => {
-                        this.emitEditorLinkLine(lineNum, column, filename, true);
-                        // do not bring user to the top of index.html
-                        // http://stackoverflow.com/questions/3252730
-                        e.preventDefault();
-                        return false;
-                    })
-                    .on('mouseover', () => {
-                        this.emitEditorLinkLine(lineNum, column, filename, false);
-                    }) as any // TODO
-            );
+            elem.empty();
+            $('<span class="linked-compiler-output-line"></span>')
```

Comment:
I think .empty / .appendTo is the best way to do this without having to cast

Reply:
Good idea. It also feels more clean this way

Reply:
Works for me!